DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -278,21 +280,23 @@ Next, look at the loop details of the *KernelArgsRestrict* kernel. You will noti
 ## Example Output
 
 ```
-Running on device: de10_agilex : Agilex Reference Platform (aclde10_agilex0)
+Running on device: pac_s10 : Intel PAC Platform (pac_f200000)
```

Comment:
There should be no more mentions to PAC boards as these are deprecated
Please compile for de10

Reply:
Completed for DE10.